core-services/secret-mirroring/_mapping.yaml

```diff
@@ -114,6 +114,96 @@ secrets:
   to:
     namespace: ci
     name: odo-secret
+- from:
+    namespace: ci-release
+    name: files-pull-secret
+  to:
+    namespace: ci-release-ppc64le
+    name: files-pull-secret
+- from:
+    namespace: ci-release
+    name: git-credentials
+  to:
+    namespace: ci-release-ppc64le
+    name: git-credentials
+- from:
+    namespace: ci-release
+    name: source
+  to:
+    namespace: ci-release-ppc64le
+    name: source
+- from:
+    namespace: ci-release
+    name: files-pull-secret
+  to:
+    namespace: ci-release-s390x
+    name: files-pull-secret
+- from:
+    namespace: ci-release
+    name: git-credentials
+  to:
+    namespace: ci-release-s390x
+    name: git-credentials
+- from:
+    namespace: ci-release
+    name: source
+  to:
+    namespace: ci-release-s390x
+    name: source
+- from:
+    namespace: ci-release
+    name: files-pull-secret
+  to:
+    namespace: ci-release-priv
+    name: files-pull-secret
+- from:
+    namespace: ci-release
+    name: git-credentials
+  to:
+    namespace: ci-release-priv
+    name: git-credentials
+- from:
+    namespace: ci-release
+    name: source
+  to:
+    namespace: ci-release-priv
+    name: source
+- from:
+    namespace: ci-release
+    name: files-pull-secret
+  to:
+    namespace: ci-release-ppc64le-priv
+    name: files-pull-secret
+- from:
+    namespace: ci-release
+    name: git-credentials
+  to:
+    namespace: ci-release-ppc64le-priv
+    name: git-credentials
+- from:
+    namespace: ci-release
+    name: source
+  to:
+    namespace: ci-release-ppc64le-priv
+    name: source
+- from:
+    namespace: ci-release
+    name: files-pull-secret
+  to:
+    namespace: ci-release-s390x-priv
+    name: files-pull-secret
+- from:
+    namespace: ci-release
+    name: git-credentials
+  to:
+    namespace: ci-release-s390x-priv
+    name: git-credentials
+- from:
+    namespace: ci-release
+    name: source
+  to:
+    namespace: ci-release-s390x-priv
+    name: source
```

Comment:
This has all been switched over to the secret bootstrap logic now...
https://github.com/openshift/release/pull/12523